fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/FallbackCursor.java

```diff
@@ -113,6 +117,9 @@ public CompletableFuture<RecordCursorResult<T>> onNext() {
                     inner.close();
                     inner = fallbackCursorSupplier.apply(lastSuccessfulResult);
                     nextResultFuture = inner.onNext();
+                    if (LOGGER.isInfoEnabled()) {
+                        LOGGER.info("fallback triggered", throwable);
```

Comment:
Is the stack trace from `throwable` important? It could be rather large, right?

Reply:
Done